### PR TITLE
Stacks CRUD interface

### PR DESCRIPTION
**Pull request by ghost**
The stack pages are able to deploy new stacks, update existing stacks and delete existing stacks. The view is implemented in an ngxTable and thus has the filter and limit features. 

list view 
screen shot 2018-07-20 at 00 18 38

Deploy Stack
screen shot 2018-07-20 at 00 14 15
One is able to edit the file in the text area as well as simply upload a .yml file. 

If the stack was unsuccessfully deployed a helpful error message (Thank you @devosray ) is displayed for the user. 
screen shot 2018-07-20 at 00 22 47

The update view automatically sends in the stack name and allows the user to change the stack file however the stack name remains fixed and unchanged 
screen shot 2018-07-20 at 00 19 36

Finally the remove button
screen shot 2018-07-20 at 00 19 29

### CODE REVIEW

**Review by CDuPlooy — APPROVED** [review]

**Review by FJMentz — APPROVED** [review]
image

**Review by devosray — CHANGES_REQUESTED** [review]
Thanks Paul, looks good.

If I can nitpick,  think the "stack successfully updated" alert can be moved down a bit. Looks a bit odd crammed at the top with that tiny white line.

image

**Review by devosray — APPROVED** [review]

**Comment by devosray** [comment]
This seems to a recurrent problem and appears on multiple pages:
image

image

And it appears correct on other pages:

image

I guess the inconsistent spacing should be a separate issue.

File: src/app/pages/stacks/stacks-view/stacks-view.component.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    );
+  }
+
+  getRowHeight(row) {
+    return (row.height = 50);
+  }
+
+  open(content, stackName: string) {
+    this.stackNameToDelete = stackName;
+    console.log(content);
```
Comment (devosray) [inline src/app/pages/stacks/stacks-view/stacks-view.component.ts]: Is there a reason this is being logged to console? Seems like debug leftovers.

**Review by devosray — CHANGES_REQUESTED** [review]
Debug output?

**Review by devosray — APPROVED** [review]

**Pull request merged**